doc/main/tbb_userguide/appendix_B.rst

```diff
@@ -4,71 +4,105 @@ Appendix B Mixing With Other Threading Packages
 ===============================================
 
 
-|full_name| can be mixed with other
-threading packages. No special effort is required to use any part of
-oneTBB with other threading packages.
+Correct Interoperability
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+|full_name| can be mixed with other threading packages. No special
+effort is required to use any part of |short_name| with other threading
+packages.
 
 
 Here is an example that parallelizes an outer loop with OpenMP and an
-inner loop with oneTBB.
+inner loop with |short_name|.
 
+.. literalinclude:: ./examples/tbb_mixing_other_runtimes_example.cpp
+   :language: c++
+   :start-after: /*begin outer loop openmp with nested tbb*/
+   :end-before: /*end outer loop openmp with nested tbb*/
 
-::
 
+The ``#pragma omp parallel`` causes the OpenMP to create a team of
+threads, and each thread executes the block statement associated with
+the pragma. The ``#pragma omp for`` indicates that the compiler should
+use the previously created thread team to execute the loop in parallel.
```

Comment:
maybe extend a little bit? smth like:
the compiler to distribute the iterations of the following loop among the threads in the existing thread team, enabling parallel execution of the loop body.

Reply:
Then it becomes
```suggestion
distribute the iterations of the following loop among the threads in the existing thread team, enabling parallel execution of the loop body.
```